app/src/main/java/fr/free/nrw/commons/Media.java

```diff
@@ -58,17 +59,22 @@ public Media createFromParcel(Parcel parcel) {
     protected String licenseUrl;
     protected String creator;
     protected ArrayList<String> categories; // as loaded at runtime?
+    protected ArrayList<String> depictions;
     protected boolean requestedDeletion;
     private Map<String, String> descriptions; // multilingual descriptions as loaded
+    private HashMap<String, String> captions;
     private HashMap<String, Object> tags = new HashMap<>();
     private @Nullable LatLng coordinates;
+    private String pageId;
```

Comment:
javadoc

Reply:
Nicolas, this change was not added my me. I think that this came due to certain changes from backend-overhaul. However I tried to understand it and add documentation for this. Please have a look at the documentation added for this in my next commit :)

Reply:
If it is not yours, don't worry too much :-)